Automattic/wp-calypso
# Olark chat: Add error boundary

#### Changes proposed in this Pull Request

* As per the discussion in p58i-ch2-p2, this PR adds a react error boundary for the Olark script.

#### Testing instructions



* Check out branch to your local calypso instance and add the following line in the `render()` function of `<OlarkChat>` component, [specifically here](https://github.com/Automattic/wp-calypso/blob/trunk/client/components/olark-chat/index.jsx#L34):
```
throw new Error( 'Test' );
```

Now, head to `/start`, the page should render and you should be able to complete signup, but the olark chat widget is expected not to be rendered. Without the changes in this PR, `/start` will not render.
